net-dns/dnsdist/dnsdist-1.1.0.ebuild

```diff
@@ -0,0 +1,134 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+EGIT_REPO_URI="https://github.com/PowerDNS/pdns.git"
+
+if [[ ${PV} = 9999 ]]; then
+	GIT_ECLASS="git-r3"
+	EXPERIMENTAL="true"
+fi
+
+inherit eutils user flag-o-matic systemd autotools ${GIT_ECLASS}
+
+DESCRIPTION="dnsdist is a highly DNS-, DoS- and abuse-aware loadbalancer."
+HOMEPAGE="http://dnsdist.org"
+
+if [[ ${PV} == 9999 ]] ; then
+	SRC_URI=""
+	S="${WORKDIR}/${P}/pdns/dnsdistdist"
+else
+	SRC_URI="https://downloads.powerdns.com/releases/${P}.tar.bz2"
+	KEYWORDS="~amd64"
+fi
+
+LICENSE="GPL-2"
+SLOT="0"
+IUSE="bindist libedit luajit +sodium +protobuf re2 dnscrypt systemd +readline"
+
+RESTRICT="readline? ( bindist )"
+
+REQUIRED_USE="
+	bindist? ( libedit )
+	libedit? ( !readline )
+	readline? ( !libedit )
```

Comment:
What I want to achieve is: when bindist is set, only libedit shall be used for compilation.